tests/test_network_consistency.py

```diff
@@ -61,7 +61,13 @@ def test_network_consistency(self, net_name, data_path, json_path):
 
         # Create model
         model = nets.__dict__[net_name](**model_params)
-        model.load_state_dict(loaded_data["model"])
+        state_dict = loaded_data["model"]
```

Comment:
I would suggest `model = getattr(nets, net_name)(**model_params)` for the previous line.